docs/executing-in-occlum.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /docs/executing-in-occlum
+---
+
+# Executing builtin-functions in Occlum
+
+The example shows how to run teaclave builtin-functions in Occlum.
+
+## Requirements
+Firstly, get the official occlum Docker image to build teaclave.
```

Comment:
Can we build teaclave in teaclave docker image and use the output in occlum image?